environment.yml

```diff
@@ -19,7 +19,7 @@ dependencies:
 - python==3.6.8
 - pytorch>=1.2.0
 - torchvision>=0.3.0
-#- pycocotools>=2.0
+- pycocotools>=2.0
```

Comment:
This line pycocotools will break the windows devops tests. How about we install cython here, but no pycocotools and rather ask the user to install manually?

Reply:
without it, the tests will still fail once we add those notebook tests to the pipeline

Reply:
True, but at least all classification and similarity tests will run. We need to find a way to also support object detection tests.

Reply:
i uncomment both for now, and in the readme instructions, tell users to uncomment one of them depending on their os.